pkgs/development/compilers/llvm/common/clang/default.nix

```diff
@@ -43,14 +43,13 @@ let
         # https://reviews.llvm.org/D51899
         (getVersionFile "clang/gnu-install-dirs.patch")
       ]
-      ++ lib.optionals (lib.versionOlder release_version "20") [
+      ++ lib.optional (lib.versionOlder release_version "20")
```

Comment:
Like in https://github.com/NixOS/nixpkgs/pull/394539#discussion_r2020105853, I think if you change this lots of people would want to change it back.

Reply:
Dropped